scripts/YaML.md

```diff
@@ -190,6 +190,49 @@ class zer_name_t(c_void_p):
 </td></tr>
 </table>
 
+#### type: fptr_typedef
+* A fptr_typedef initiates the creation of a C/C++ `typedef` of a function pointer declaration in the specification
+* A fptr_typedef requires the following scalar fields: {`desc`, `name`, `return`}
+  - `desc` will be used as the typedef's description comment
+  - `name` must be a unique ISO-C standard identifier, start with `$` tag, be snake_case and end with `_t`
+  - `return` must be an ISO-C standard data type
+* A function pointer typedef can optionally accept the following sequence of mappings: {`params`}
+  - A param requires the following scalar fields: {`desc`, `type`, `name`}
+  - `desc` will be used as param's description comment
+  - `name` must be a unique ISO-C standard identifier
+  - `type` must be ISO-C standard data type
+
+<table>
+<tr><th>YaML</th><th>C</th><th>C++</th></tr>
+<tr><td>
+
+```yml
+type: fptr_typedef
+desc: "A brief description..."
+name: $x_callback_t
+return: "void"
+params:
+    - type: void*
+      name: pParams
+      desc: "Brief description of param"
+```
+</td>
+<td>
+
+```c
+/// @brief A brief description...
+typedef void(zer_callback_t)(void* pParams);
```

Comment:
Can we use `$x` here and below so we don't have to update the code when we change the `zer` prefix to `ur`? Or will that not work inside this code snippet?

Reply:
It wont work here, this code is a snippet.